src/java.base/share/classes/java/nio/file/Path.java

```diff
@@ -516,6 +516,110 @@ default Path resolve(String other) {
         return resolve(getFileSystem().getPath(other));
     }
 
+    /**
+     * Resolves a path against this path, and then iteratively resolves any
+     * additional paths.
+     *
+     * <p> If {@code more} does not specify any {@code Path}s,
+     * then the result is {@link #resolve(Path) resolve(first)}. If
+     * {@code more} specifies one or more {@code Path}s, then each non-empty
+     * {@code Path}, including {@code first}, is resolved iteratively against
+     * this {@code Path}.
```

Comment:
The second paragraph has the same issue as before, it says "resolved iteratively against this Path" but that isn't correct. Same issue in the description of the "more" method.  The first paragraph is correct, and I think we'll need to re-phrase the second paragraph to accurate specify how the iterative resolution works.

Reply:
0fa126fd2d3861c56b85093e8a2483afa1ad9ef6 attempts to improve the aforementioned verbiage.

Reply:
> 0fa126f  attempts to improve the aforementioned verbiage.

The wording in the latest (db4d9bd6) looks good.